easybuild/framework/easyblock.py

```diff
@@ -455,12 +455,16 @@ def fetch_patches(self, patch_specs=None, extension=False, checksums=None):
         All patches will be checked if a file exists (or can be located)
         """
         if patch_specs is None:
-            patch_specs = self.cfg['patches']
+            post_install_patches = self.cfg['postinstallpatches']
+            patch_specs = self.cfg['patches'] + post_install_patches
+        else:
+            post_install_patches = []
```

Comment:
I have a minor suggestion for readability:
```
        post_install_patches = []

        if patch_specs is None:
            # if no patch_specs are specified, add all pre-install and post-install patches
            post_install_patches = self.cfg['postinstallpatches']
            patch_specs = self.cfg['patches'] + post_install_patches
```

Reply:
fixed in 9896bbb